streaming/java/streaming-runtime/src/main/java/org/ray/streaming/runtime/core/master/resourcemanager/ResourceManager.java

```diff
@@ -12,9 +12,9 @@
 public interface ResourceManager {
 
   /**
-   * Get all registered container as a list.
+   * Get all registered container as a LIST.
```

Comment:
ditto.

There are too many these changes. Please check by yourself or directly revert the commit and redo meaningful actions.